Add support for sixth player

nwai90 opened the pull request:
**Ref:** https://github.com/bafolts/terraforming-mars/issues/801

Seems to work fine when tested locally. Issues like having too few corporations in the base game to deal out 3 to each of the 6 players have already been previously handled.

<img width="801" alt="6p_1" src="https://user-images.githubusercontent.com/2408094/84569985-e4110900-adbc-11ea-9e0c-5ee95254a55f.png">

<img width="362" alt="6p_2" src="https://user-images.githubusercontent.com/2408094/84569986-e5dacc80-adbc-11ea-9b4b-4c4227ef8f54.png">

<img width="277" alt="Screenshot 2020-06-14 at 4 32 13 AM" src="https://user-images.githubusercontent.com/2408094/84578476-27d53400-adf8-11ea-9834-9d16688e4871.png">

<img width="91" alt="Screenshot 2020-06-14 at 4 28 31 AM" src="https://user-images.githubusercontent.com/2408094/84578477-299ef780-adf8-11ea-9bd8-885744f307cf.png">

<img width="333" alt="Screenshot 2020-06-14 at 4 28 09 AM" src="https://user-images.githubusercontent.com/2408094/84578478-2dcb1500-adf8-11ea-9ef8-669251d3b821.png">

AlexBurkov commented:
When I added sixth player here I had to add images for cube and ship for colony. Can you check it in your PR? In other cases it was ok because we played it a lot of times.

nwai90 commented:
> When I added sixth player here I had to add images for cube and ship for colony. Can you check it in your PR? In other cases it was ok because we played with six players a lot of times. And I used this color #800080. I think it's more light color:)
> ![image](https://user-images.githubusercontent.com/12098651/84571519-56bdbc80-ad9c-11ea-816c-cd3822452e6e.png)

Added the new assets!
